### PR TITLE
CDK: update matrix

### PR DESCRIPTION
**Pull request by kmurphypolygon**
Version updates https://polygon.atlassian.net/browse/DOCS-139

### CODE REVIEW

File: docs/cdk/version-matrix.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The table below shows the version compatibility for CDK releases and related com
     <td>CDK Elderberry 2 Release </td>
     <td>9</td>
     <td><a href="https://github.com/0xPolygonHermez/zkevm-node/releases/tag/v0.6.4">v0.6.4 Elderberry 2</a></td>
-    <td><a href="https://github.com/0xPolygon/cdk-validium-node/releases/tag/v0.6.4%2Bcdk.3">0.6.4+cdk.3</a> </td>
+    <td><a href="https://hub.docker.com/layers/0xpolygon/cdk-validium-node/0.6.4-cdk.5/images/sha256-0b56130f03b687833daa561a82088715d3e1fbbcba6de4eb122727c40b181987?context=explore">0.6.4+cdk.5</a> </td>
```
Comment (hhjmmmmmmmm) [inline docs/cdk/version-matrix.md:32]: 0x5dA1CD01B5d373b1B9Ca25d46660b51708D1dFc80x5dA1CD01B5d373b1B9Ca25d46660b51708D1dFc8

**Review by Sharonbc01 — CHANGES_REQUESTED** [review]
Please refer to the slide I see that the Prover version for Fork ID 7 should be 0.0.6

**Comment by kmurphypolygon** [comment]
> Please refer to the slide I see that the Prover version for Fork ID 7 should be 0.0.6

Please check now.

**Comment by kmurphypolygon** [comment]
> > Please refer to the slide I see that the Prover version for Fork ID 7 should be 0.0.6
> 
> Please check now.

Oh, it's actually the DAC that's updated, not prover. Should be oK.

**Review by Sharonbc01 — APPROVED** [review]

**Pull request merged**